### DISCUSSION TITLE
Unable to login with mobile 2FA

### DISCUSSION THREAD

**Discussion by orbisvicis** [community/community#34355] [Mobile] [answered]
After installing, logging-in and registering the mobile app, I am unable to use the mobile 2FA to log in via the browser. I've tried multiple browsers until I was locked out, and the error message is always "digits don't match". I'm on a fast network and type in the code very fast; a few times at least under a second.

This is the process:
* Open browser
* Enter username & password
* Two-digit number displayed
* Enter two-digit number in mobile app
* Login fails; digits don't match.

**Comment by Iron-Ham** [comment]
Hi @orbisvicis

Thank you for reporting. 
Is this on iOS or Android, and could you let me know what version of the app you're using?

**Reply by orbisvicis** [reply]
Pixel 4, Android 13, app version 1.81.0.

**Reply by coingint** [reply]
Thank you

**Comment by orbisvicis** [comment, marked as answer]
Moto G7 Power, Lineage OS 19.1, app version 1.98.0.

2FA via the app is now working.

**Comment by agimbrac** [comment]
> Moto G7 Power, Lineage OS 19.1, app version 1.98.0.
> 
> 
> 
> 2FA via the app is now working.

**Comment by anero1** [comment]
having issues logging into my new device with 2FA.  it says the code is not correct, but it is on my phone

**Comment by m775115559-rgb** [comment]
> Moto G7 Power, Lineage OS 19.1, app version 1.98.0.
> 
> 2FA via the app is now working.

**Comment by m775115559-rgb** [comment]
> After installing, logging-in and registering the mobile app, I am unable to use the mobile 2FA to log in via the browser. I've tried multiple browsers until I was locked out, and the error message is always "digits don't match". I'm on a fast network and type in the code very fast; a few times at least under a second.
> 
> This is the process:
> * Open browser
> * Enter username & password
> * Two-digit number displayed
> * Enter two-digit number in mobile app
> * Login fails; digits don't match.

**Comment by rigras** [comment]
Hello, im having the same problem as @orbisvicis 

 I was working on my work computer for my company in which I setted up my personal account. I was able to use the account but i no longer have that computer. 

Therefore I started to use my personal computer in which I already worked before with my git account( I even have access to my repos via the cli ), but when im trying to login to the web browser with 2fa authentication im having the same problem. 

This is the process:

Open browser
Enter username & password
Two-digit number displayed
Enter two-digit number in mobile app
Login fails; digits don't match.

@Iron-Ham  how ere you able to solve it ?

**Reply by Iron-Ham** [reply]
I recommend you open a new discussion for support from the team.
